styles/style.scss

```diff
@@ -0,0 +1,260 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+  font-family: "Karla", sans-serif;
+  background: hsl(204, 43%, 93%);
+}
+footer {
+  display: flex;
+  justify-content: center;
+  font-size: 1.5rem;
+}
+.grid-container {
+  display: grid;
+  grid-template-columns: 0.9fr 0.5fr 0.5fr 0.9fr;
+  grid-template-rows: 0.8fr 0.5fr 0.7fr 0.5fr;
+  max-height: 100vh;
+  .bg1 {
+    grid-column-start: 2;
```

Comment:
I see that a lot of the styles are being reused by being duplicated. You can avoid this by setting a common class between all that is common and extending that with custom styles. Something like this

`.common {
  common-styles
 . particular {
    particular-stles
}
}
`